pkg/image/utilities/utilities.go

```diff
@@ -485,14 +495,14 @@ for i in "${linux_images[@]}"; do
     fi
     case $i in
     */*)
-        image_name="${reg}/${i}"
+        image_name="${target_registry}/${i}"
         ;;
     *)
-        image_name="${reg}/rancher/${i}"
+        image_name="${target_registry}/rancher/${i}"
         ;;
     esac
 
-    docker tag "${i}" "${image_name}${arch_suffix}"
+    docker tag "${source_registry}${i}" "${image_name}${arch_suffix}"
```

Comment:
```suggestion
    docker tag "${source_registry}/${i}" "${image_name}${arch_suffix}"
```

if `--registry` is expected to not have a `/` at the end of it, so is `--source-registry`